### PR TITLE
Remove Study ID column from Study sheet

### LINKED ISSUE #1044: Remove the Study ID column from the Study sheet
### Inspiration

Since animals can belong to multiple studies, using the `Study.code` field as a prefix in automatic human-readable ID generation is not ideal and needs to be re-thought.

### Description

Remove the Study ID column on the Study page (and `code` as a field from the `Study` model).

### Alternatives

1. Change the concept to a "Submission ID" (though).
2. Append (sorted) prefixes(/IDs/codes) together when an animal belongs to multiple studies and keep the `Study.code` field.
3. Only remove the column from the input file format and keep the field until we decide whether we can/want-to use it for automatic ID generation.

### Dependencies

- #1037

### Comment

None

-----

### Assumptions

1. List of assumptions that the code will not explicitly address/check
4. E.g. We will assume input is correct (explaining why there is no validation)

### Limitations

1. A list of things this work will specifically not do
2. E.g. This feature will only handle the most frequent use case X

### Affected Components

- change: File path or DB table ...
- add: Environment variable or server setting
- delete: External executable or cron job

### Requirements

- [ ] 1. List of numbered conditions to be met for the feature
- [ ] 2. E.g. Every column/row must display a value, i.e. cannot be empty
- [ ] 3. Numbers for reference & checkboxes for progress tracking

#### Interface Change description

None provided

#### Code Change Description

None provided

#### Tests

- [ ] 1. A description of at least one test for each requirement above.
- [ ] 2. E.g. Test for req 2 that there's an exception when display value is ''
- [ ] 3. Numbers for reference & checkboxes for progress tracking

### PR DESCRIPTION
**Pull request by hepcat72**
## Summary Change Description

See affected issue(s).

## Affected Issues/Pull Requests

- Resolves #1044
- Merges into PR #1084
- Next PR: #1113

## Review Notes

See comments in-line.

## Checklist

This pull request will be merged once the following requirements are met.  The
author and/or reviewers should uncheck any unmet requirements:

- Review requirements
  - Minimum approvals: 1 
  - No changes requested
  - All blocking issues resolved by reviewers
  - Specific reviewers: @__add_username_here__
    
  - Review period: 2 days 
- Associated issue/pull request requirements:
  
  - [x] All requirements in affected issues marked "resolved" are satisfied
  - [x] All required pull requests are merged *(or none)*
- Basic requirements
  
  - [x] [All linters pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting)
  - [x] [All tests pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)
  - [x] All conflicts resolved
- Overhead requirements
  
  - [x] [New/changed method tests implemented/updated *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
  - [ ] [Updated *Unreleased* section of `changelog.md` *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/changelog.md)
  - [x] [Migrations created & committed *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)

### CODE REVIEW

File: DataRepo/models/study.py [hunk 1]
```diff
@@ -9,6 +9,7 @@ class Study(models.Model):
     atleast2chars = MinLengthValidator(2)
 
     id = models.AutoField(primary_key=True)
+    # TODO: Remove the code field
```
Comment (lparsons) [inline DataRepo/models/study.py:12]: Q: Should this remain a TODO comment or be done in this (or a later) PR?

**Review by lparsons — APPROVED** [review]
Looks good, just check the TODO comment. It seems like the code field could be removed now, but perhaps it breaks too may things atm?

File: DataRepo/models/study.py (hunk 1 above)
Comment (hepcat72) [reply DataRepo/models/study.py:12]: I could try it.  I was just avoiding dealing with a migration.  It probably would go off without a hitch though.

Comment (hepcat72) [reply DataRepo/models/study.py]: Removed.

**Pull request merged**